src/plugins/kibana_usage_collection/server/collectors/event_loop_delays/track_delays.ts

```diff
@@ -9,7 +9,7 @@
 import { takeUntil, finalize, map } from 'rxjs/operators';
 import { Observable, timer } from 'rxjs';
 import type { ISavedObjectsRepository } from '@kbn/core/server';
-import type { EventLoopDelaysMonitor } from '@kbn/core/server';
+import type { IEventLoopDelaysMonitor, IntervalHistogram } from '@kbn/core/server';
```

Comment:
I thought about changing the import to import directly from the `@kbn/core-metrics-server` package, since `kibana_usage_collection` is owned by core and wouldn't require another team's review. However, I thought it best to keep some consistency and import from core for now.